src/modlunky2/ui/trackers/category.py

```diff
@@ -21,11 +21,85 @@
 ICON_PATH = BASE_DIR / "static/images"
 
 
+class CategoryModifiers(ttk.LabelFrame):
+    def __init__(self, parent, modlunky_config: Config, *args, **kwargs):
+        super().__init__(parent, *args, **kwargs)
+        self.parent = parent
+
+        self.modlunky_config = modlunky_config
+
+        self.always_show_modifiers = tk.BooleanVar()
+        self.always_show_modifiers.set(
+            modlunky_config.trackers.category.always_show_modifiers
+        )
+
+        widgets = []
+        widgets.append(
+            ttk.Checkbutton(
+                self,
+                text="Always Show",
+                variable=self.always_show_modifiers,
+                onvalue=True,
+                offvalue=False,
+                command=self.toggle_always_show_modifiers,
+            )
+        )
+        widgets.append(ttk.Separator(self))
+
+        # Starting Category Exclusion
+
+        ## This does not have any effect on the actual tracking logic, rather,
+        ## like always_show_modifiers, it only has an effect on the actual
+        ## text returned.
+
+        ## The checkboxes imply *inclusion*, not exclusion, so the
```

Comment:
While seemingly so, the logic is built around excluding it from visualization rather than including it in the tracking logic, so the tracking logic can be identical regardless if you're actually running No/NG/Pacifist. I understand the tracking logic is intricate, which is why I went with the approach of using exclusions and just doing a simple translation near the UI layer.

Reply:
Makes sense